jenkinsci/blueocean-plugin
# [JENKINS-47929] Allow extensible run details tabs

# Description

See [JENKINS-47929](https://issues.jenkins-ci.org/browse/JENKINS-47929).
Reviving [PR#1541](https://github.com/jenkinsci/blueocean-plugin/pull/1541).

**Unit/Acceptance Tests**
AFAICT the existing tests provide enough coverage for this change as the default actions (changes, tests, artifacts) have been modified to use the new extension point.

**Manual Test Instructions**
- Build and run the changes to verify that the changes, tests and artifacts action tabs still work

# Submitter checklist

# Reviewer checklist
